src/core/prometheus.cc

```diff
@@ -660,9 +660,9 @@ class metrics_handler : public handler_base  {
 
     future<std::unique_ptr<httpd::reply>> handle(const sstring& path,
         std::unique_ptr<httpd::request> req, std::unique_ptr<httpd::reply> rep) override {
-        sstring metric_family_name = req->get_query_param("name");
-        bool show_help = req->get_query_param("help") != "false";
+        sstring metric_family_name = req->get_query_param("__name__");
         bool prefix = trim_asterisk(metric_family_name);
+        bool show_help = req->get_query_param("__help__") != "false";
```

Comment:
We also need to document this "`help`" that now changed to "`__help__`".
Are we sure that no existing user, script, tool, or whatever, needs the old names "name" and "help"?

Reply:
definitely, I'm rewriting the section for V2